### DISCUSSION TITLE
Rename a test suite

### DISCUSSION THREAD

**Discussion by denissignoretto** [Kong/insomnia#6051] [Ideas]
Now, Insomnia v.2023.2.2, I didn't find a way to rename a test suite.

I'd like to know whether such functionality exists or not.

Thanks,
Denis

**Comment by denissignoretto** [comment]
I've discovered that it's possible reanem test suite double clicking on the title of the test suite.

In other functionalities (like requests) there is a drop "Reneame" down menu.

I think it should be desiderable having both possibilities: "rename" from drop down menu an double clicking on test suite title as well.

**Comment by kobenguyent** [comment]
I think right now, wherever you hover over the test suite for example, the cursor changes to hand icon and I suppose we know that okay we need to click it.
